modules/serverless-rn-1-7-1.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-7-1_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.7.1
+
+[id="new-features-1-7-1_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.13.3.
+* {ServerlessProductName} now uses Knative Serving Operator 0.13.3.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.13.2.
+* {ServerlessProductName} uses Knative Eventing 0.13.0.
+* {ServerlessProductName} now uses Knative Eventing Operator 0.13.3.
+
+[id="fixed-issues-1-7-1_{context}"]
+== Fixed issues
+
+* In {ServerlessProductName} 1.7.0, there was a bug that caused {product-title} routes to be reconciled continuously when this was not required. This bug is fixed in {ServerlessProductName} 1.7.1.
```

Comment:
I'd s/there was a bug that caused {product-title} routes to be/{product-title} routes were